src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParseException.cs

```diff
@@ -1,27 +1,52 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Globalization;
```

Comment:
(nit) I haven't checkout out the PR, but sometimes VS adds these usings and we end up not using them. If we're not using System.Globalization, can you delete it on your next pass here?

Reply:
Fixed it.